lib/active_admin/filters/active.rb

```diff
@@ -8,13 +8,21 @@ class Active
 
       def initialize(resource_class, params)
         @resource_class = resource_class
-        @params = params.respond_to?(:to_unsafe_h) ? params.to_unsafe_h : params
+        @params = normalize_params(params)
         @scope = humanize_scope
         @filters = build_filters
       end
 
       private
 
+      def normalize_params(params)
+        if params.kind_of?(HashWithIndifferentAccess)
```

Comment:
There's much more usages of `is_a?` than `kind_of?` in the codebase (35 vs 3 if I counted corrected), so maybe use `is_a?` to keep it more consistent.